apis/v1alpha2/httproute_types.go

```diff
@@ -282,6 +283,23 @@ const (
 	HeaderMatchImplementationSpecific HeaderMatchType = "ImplementationSpecific"
 )
 
+// HTTPHeaderName is the name of an HTTP header.
+//
+// Valid values include:
+//
+// * Authorization
```

Comment:
put "quotes" to make it clear these are strings

Reply:
Updated, thanks!